lib/DriverTool/autolink_extract_main.cpp

```diff
@@ -229,6 +252,13 @@ int autolink_extract_main(ArrayRef<const char *> Args, const char *Argv0,
 
   std::vector<std::string> LinkerFlags;
 
+  // Keep track of whether we've already added the common
+  // Swift libraries that ususally have autolink directives
+  // in most object fiels
+  std::unordered_map<std::string, bool> CommonAddedSwiftLibraries = {{"-lswiftSwiftOnoneSupport", false},
+                                                                    {"-lswiftCore", false},
```

Comment:
Technically, `-lswiftCore` should be added by the driver (and this tool is only ever invoked by the swift driver) ... so we could even eliminate it entirely.

Reply:
I'd like to put this change in a release asap so let's just de-duplicate it for now to be safe and then examine the duplication with the driver, too. Because it'd probably be better to remove this at the place where the directives get inserted, instead of completely removing here.